commitizen.config.js

```diff
@@ -0,0 +1,43 @@
+const types = [
+  {
+    value: 'feat',
+    name: 'feat:     A new feature'
+  },
+  {
+    value: 'fix',
+    name: 'fix:      A bug fix'
+  },
+  {
+    value: 'docs',
+    name: 'docs:     Documentation only changes'
+  },
+  {
+    value: 'refactor',
+    name: 'refactor: A code change that neither fixes a bug nor adds a feature'
+  },
+  {
+    value: 'test',
+    name: 'test:     Adding missing tests'
+  },
+  {
+    value: 'chore',
+    name: 'chore:    Changes to the build process or auxiliary tools\n            and libraries such as documentation generation'
+  },
+  {
+    value: 'revert',
+    name: 'revert:   Revert to a commit'
+  }
+];
+
```

Comment:
@jlegrone I'm good with this for now, we may want something like `style` as well